CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * `RealmObject.asFlow` will throw `UnsupportedOperationException` instead of `IllegalStateException` if called on a live or dynamic object in a write transaction or in a migration.
 * `RealmObject.asFlow` will throw `UnsupportedOperationException` instead of `IllegalStateException` if called on a live or dynamic object in a write transaction or in a migration.
 * Removed `RealmObject.delete()` and `RealmResults.delete()`. All objects, objects specified by queries and results must be delete through `MutableRealm.delete(...)` and `DynamicMutableRealm.delete(...).
+* Removed default empty schema argument for `RealmConfiguration.Builder(schema = ... )` and `SyncConfiguration.Builder(..., schema= ... )` as all configuraitons require a non-empty schema.
```

Comment:
Do we actually validate that the schemas are not empty?

Reply:
No, but we decided to leave it as is for now: https://mongodb.slack.com/archives/CSXQ43YFK/p1646321007587149?thread_ts=1646319193.809879&cid=CSXQ43YFK